homeassistant/components/dsmr_reader/sensor.py

```diff
@@ -35,14 +33,9 @@ def __init__(self, topic):
         self._topic = topic
 
         self._name = self._definition["name"]
-        self._unit_of_measurement = (
-            self._definition["unit"] if "unit" in self._definition else ""
-        )
-        self._icon = self._definition["icon"] if "icon" in self._definition else None
-        self._transform = (
-            self._definition["transform"] if "transform" in self._definition else None
-        )
-
+        self._unit_of_measurement = self._definition.get("unit", None)
```

Comment:
`None` is the default value returned if the key is missing in the dict. We can remove that argument.